### PR TITLE
Manifests and metadata for version 3.57.0-1

### PR DESCRIPTION
**Pull request by bobotimi**
Generated metadata and manifests for Nexus Repository 3.57.0-1 Open shift operator
The outcome of running ./scripts/new_version.sh bundle 3.57.0-1  `<operator-image-url> <app-image-url>`

### CODE REVIEW

File: deploy/olm-catalog/nxrm-operator-certified/3.57.0-1/manifests/nxrm-operator-certified.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,432 @@
...
+                "enabled": true,
+                "secretName": "nexus-tls"
+              }
+            },
+            "nexus": {
+              "dockerPort": 5003,
+              "env": [
+                {
+                  "name": "INSTALL4J_ADD_VM_PARAMS",
+                  "value": "-Xms2703M -Xmx2703M -XX:MaxDirectMemorySize=2703M -XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap"
```
Comment (mobiuscog) [inline deploy/olm-catalog/nxrm-operator-certified/3.57.0-1/manifests/nxrm-operator-certified.clusterserviceversion.yaml:53]: Do we need maxDirectMemory if we're not using Orient ?

File: deploy/olm-catalog/nxrm-operator-certified/3.57.0-1/manifests/nxrm-operator-certified.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,432 @@
...
+        }
+      ]
+    capabilities: Basic Install
+    categories: "Integration & Delivery"
+    description: |-
+      Nexus Repository is the central source of control to efficiently manage all binaries
+      and build artifacts across your DevOps pipeline.
+    containerImage: registry.connect.redhat.com/sonatype/nxrm-operator-certified@sha256:2ff1738e961e6f4f883c10d421a6730237eefbcc052dfb28623395e42638e4ca
+    repository: https://github.com/sonatype/operator-nxrm3
+    createdAt: 2020-07-17
```
Comment (mobiuscog) [inline deploy/olm-catalog/nxrm-operator-certified/3.57.0-1/manifests/nxrm-operator-certified.clusterserviceversion.yaml:135]: Should this be updated ?

File: deploy/olm-catalog/nxrm-operator-certified/3.57.0-1/manifests/nxrm-operator-certified.clusterserviceversion.yaml [hunk 3]
```diff
@@ -0,0 +1,432 @@
...
+    Version control systems and package registries do not scale when managing proprietary,
+    open source, and third-party components.
+    Organizations need a central binary and build artifact repository to manage dependencies
+    across the entire software supply chain.
+
+    ## Limitations
+
+    High Availability Clustering (HA-C) is not supported for Nexus Repository Pro for OpenShift.
+
+    This operator will be released on a quarterly basis.
```
Comment (mobiuscog) [inline deploy/olm-catalog/nxrm-operator-certified/3.57.0-1/manifests/nxrm-operator-certified.clusterserviceversion.yaml:178]: Is this line still correct ?

**Review by mobiuscog — APPROVED** [review]
A few questions

File: deploy/olm-catalog/nxrm-operator-certified/3.57.0-1/manifests/nxrm-operator-certified.clusterserviceversion.yaml (hunk 1 above)
Comment (bobotimi) [reply deploy/olm-catalog/nxrm-operator-certified/3.57.0-1/manifests/nxrm-operator-certified.clusterserviceversion.yaml:53]: Oh, this is still the orient operator

File: deploy/olm-catalog/nxrm-operator-certified/3.57.0-1/manifests/nxrm-operator-certified.clusterserviceversion.yaml [hunk 4]
```diff
@@ -0,0 +1,432 @@
...
+      Increase DevOps scalability with integrations to the most popular build and deployment tools
+
+    Version control systems and package registries do not scale when managing proprietary,
+    open source, and third-party components.
+    Organizations need a central binary and build artifact repository to manage dependencies
+    across the entire software supply chain.
+
+    ## Limitations
+
+    High Availability Clustering (HA-C) is not supported for Nexus Repository Pro for OpenShift.
```
[the remark this answers is not in the document]
Comment (bobotimi) [reply deploy/olm-catalog/nxrm-operator-certified/3.57.0-1/manifests/nxrm-operator-certified.clusterserviceversion.yaml:176]: This operator is the one for orient

File: deploy/olm-catalog/nxrm-operator-certified/3.57.0-1/manifests/nxrm-operator-certified.clusterserviceversion.yaml (hunk 3 above)
Comment (bobotimi) [reply deploy/olm-catalog/nxrm-operator-certified/3.57.0-1/manifests/nxrm-operator-certified.clusterserviceversion.yaml:178]: Maybe that was the intention when this was created.

File: deploy/olm-catalog/nxrm-operator-certified/3.57.0-1/manifests/nxrm-operator-certified.clusterserviceversion.yaml (hunk 2 above)
Comment (bobotimi) [reply deploy/olm-catalog/nxrm-operator-certified/3.57.0-1/manifests/nxrm-operator-certified.clusterserviceversion.yaml:135]: Hmm, that date is coming from the template. Although it looks like all previous releases didn't. Nevertheless, I can update it an regenerate

Comment (bobotimi) [reply deploy/olm-catalog/nxrm-operator-certified/3.57.0-1/manifests/nxrm-operator-certified.clusterserviceversion.yaml]: https://github.com/sonatype/operator-nxrm3/pull/59/commits/6bf4be9505e1c0db5546cdeb68221869cf057436

**Pull request merged**